VoltAir/Game.cpp

```diff
@@ -335,6 +335,14 @@ void Game::revealAchievement(const QString& name, bool immediately) {
     mPlayerProfile.revealAchievement(name, immediately);
 }
 
+QString Game::getValidLetter(const double randomNumber)
```

Comment:
@yorugac How does this fit with your implementation of fetching random letters for the gameplay?